src/administration/models.py

```diff
@@ -112,3 +113,23 @@ class Meta:
 
     def __str__(self):
         return self.email
+
+class WantToHelp(models.Model):
+    name=models.CharField(max_length=225)
+    email=models.EmailField (max_length=255)
+    description=models.TextField()
+
+
+
+class OpportunityCategory(models.Model):
+    name=models.CharField(max_length=225)
+    slug=models.SlugField()
+    opportunities=models.ManyToManyField(Opportunity)
+    created=models.DateTimeField( auto_now_add=True)
+    updated=models.DateTimeField( auto_now=True)
+
+    def save(self, *args, **kwargs):
```

Comment:
it's not necessary to slugify on save, slug must be required and therefore can't be null